app/main.py

```diff
@@ -1,6 +1,41 @@
+import os
+import requests
+
+
+API_KEY = os.getenv("API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    if API_KEY is not None:
```

Comment:
Consider adding a check to ensure that `API_KEY` is not an empty string, as this would also cause authentication to fail.